.gitignore

```diff
@@ -8,12 +8,17 @@ release-packages/
 release-gcp/
 release-images/
 node_modules
-vendor/modules
-node-*
 /plugins
 /lib/coder-cloud-agent
 .home
 coverage
 **/.DS_Store
+
+# Code packages itself here.
+/lib/vscode-reh-web-linux-x64/
```

Comment:
question: are there other `vscode-reh-web-*` packages we need to ignore?

Reply:
Ah whoops good point.  This should be a wildcard.